core/src/main/java/io/grpc/internal/OobChannel.java

```diff
@@ -180,6 +180,11 @@ public Object getInternalSubchannel() {
         public PickResult pickSubchannel(PickSubchannelArgs args) {
           return result;
         }
+
+        @Override
+        public String toString() {
+          return "Subchannel picker with pick result: " + result;
```

Comment:
Better to use toStringHelper in all cases to produce standardized outputs.

Reply:
Done.